base/client.jl

```diff
@@ -71,6 +72,19 @@ function repl_cmd(cmd, out)
         ENV["OLDPWD"] = new_oldpwd
         println(out, pwd())
     else
+        @windows_only begin
+            if lowercase(shell[1]) == "cmd"
+                cmdbuf = IOBuffer()
+                print_cmdshell_escaped(cmdbuf, cmd.exec...)
+                cmd.exec = ["cmd"; "/s /c \"$(bytestring(cmdbuf.data))\""]
+                cmd.verbatimargument = true
+            elseif lowercase(shell[1]) == "powershell"
+                if length(cmd.exec) >= 2 && cmd.exec[2] == "--%"
+                  cmd.exec[2] = "\"--%\""
```

Comment:
...which I guess is more or less what libuv already does by default. Sorry about the noise.